pydoro/pydoro_tui.py

```diff
@@ -13,13 +12,8 @@
 from pydoro.pydoro_core.tomato import Tomato
 from pydoro.pydoro_core.util import every
 
-import configparser
-import os
-
-
 tomato = Tomato()
```

Comment:
We can have this as `None` then fill it up after parsing arguments and configs.

Reply:
Let me know if this doesn't work.

Reply:
The configs are loaded at the instantiation of the `Tomato` object. The problem I'm actually facing is in the instantiation of the `State` objects. For example in this `SmallBreakState`:

https://github.com/JaDogg/pydoro/blob/a5373036ecb1cdeae94bf69aa062c7c28516a3ba/pydoro/pydoro_core/tomato.py#L329-L334


The problem is that here the `time period` is defined _before_ we assign the `Tomato` object (which ultimately would contain the configuration option of the time).

While writing this comment I came up with an idea: let's make the `Tomato` assignment mandatory on the `State` objects more or less changing `def __init__(self, time_period=0, tomato=None):` to `def __init__(self, tomato):`. This would not break any existing code because (as far as I saw) all instantiations of `State` objects assign a `Tomato` object.

Reply:
Yeah getting rid of the time_period make sense to me :)